File: gto-support-okta/src/main/java/com/okta/authfoundation/claims/DefaultClaimsProviderInternals.java
package com.okta.authfoundation.claims;

import androidx.annotation.RestrictTo;

import kotlinx.serialization.json.Json;
import kotlinx.serialization.json.JsonObject;

@RestrictTo(RestrictTo.Scope.LIBRARY)
class DefaultClaimsProviderInternals {
    @SuppressWarnings("KotlinInternalInJava")
    static DefaultClaimsProvider create(JsonObject claims, Json json) {
        return new DefaultClaimsProvider(claims, json);
    }
}

File: gto-support-okta/src/main/kotlin/com/okta/authfoundation/claims/DefaultClaimsProviderInternals.kt
package com.okta.authfoundation.claims

import kotlinx.serialization.json.Json
import kotlinx.serialization.json.JsonObject

@Suppress("ktlint:standard:function-naming")
internal fun DefaultClaimsProvider(claims: JsonObject, json: Json) = DefaultClaimsProviderInternals.create(claims, json)
